### DISCUSSION TITLE
Ability to add custom classes to reduce css

### DISCUSSION THREAD

**Discussion by SergiArias** [orestbida/cookieconsent#454] [Ideas]
Hi!
Nowadays tailwind is quite popular, and I try to reduce my css as much as possible. So, I think it would be a nice feature to add the ability to define custom classes for buttons etc... and in this way reduce significantly the css as most of this classes are already defined for the website.

**Comment by dashdanilo** [comment]
@orestbida Why not improve css, use scss, apply classes in creation. Another point, why choose to create all html via javascript? If it is necessary to change something, we have to change the core of the project, which makes its adoption quite difficult, we have already tried to use it, but these were the problems that arose, are you thinking of improving, following another path?

**Reply by orestbida** [reply]
You may try [v3](https://cookieconsent.orestbida.com/), which adopts the same concept as v2. It also offers a [`core` version](https://github.com/orestbida/cookieconsent/tree/v3.0-beta/dist/core) (no gui).
